### PR TITLE
Implements show number of contributors

### PR DESCRIPTION
**Pull request by ryugoo**
## Issue
- #181

## Overview (Required)
- Implements show number of contributors
- Remove unnecessary import

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -35,6 +35,7 @@
     <string name="sponsors">スポンサー</string>
     <!-- Contributors -->
     <string name="contributors">コントリビュータ</string>
+    <string name="people">(%1$d人)</string>
```
Comment (konifar) [inline app/src/main/res/values-ja/strings.xml]: Can you add `contributors_` prefix?

File: app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java [hunk 2]
```diff
@@ -47,11 +46,11 @@
 
     private Adapter adapter;
 
-    public static ContributorsFragment newInstance() {
-        return new ContributorsFragment();
+    public ContributorsFragment() {
     }
 
-    public ContributorsFragment() {
+    public static ContributorsFragment newInstance() {
```
Comment (jmatsu) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java]: Could you please align the order of static methods and member methods with other classes? Thanks.

Comment (ryugoo) [inline app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/ContributorsFragment.java]: OK. I will fix it!

As for the question, is there a code style definition somewhere?
I refer to "codeStyleSettings.xml", but when I use the code format the constructor and static method will be swapped.

**Comment by konifar** [comment]
LGTM!

**Pull request merged**